### ISSUE TITLE
Multiple entries for the same album

### ISSUE THREAD

**Issue by jaragunde** [flaviotordini/musique#16]
I have several compilation albums with music from multiple artists, typically a different one per track. When my library was imported into Musique some albums were wrong, creating one entry per artist and album. I think that the mp3 albums were imported correctly and the ogg ones weren't.

BTW, thanks for your great application. It's a very beautiful player, we could use more software like that in Linux ;-)

**Comment by flaviotordini** [comment]
Yes, I'm aware of the bug. Will try to fix this for the next release (which should come soon)

**Comment by SangeloDev** [comment]
I'd like to bring awareness to this issue, as it is still open, and I have a similar issue with a mix of mp3 and flac files. I've got them inside my library as separate directories like this:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/47925793/174663499-f33461c5-afb4-4568-bda3-8f67c1c12e50.png">
Would be really happy to see some progress on this bug (if you could call it that). One suggestion I could leave here is an option to blacklist individual folders inside the library. Thanks for the consideration in advance!
